lib/specinfra/helper/detect_os/debian.rb

```diff
@@ -1,7 +1,7 @@
 class Specinfra::Helper::DetectOs::Debian < Specinfra::Helper::DetectOs
   def detect
     if (debian_version = run_command('cat /etc/debian_version')) && debian_version.success?
-      distro  = nil
+      distro  = 'debian'
```

Comment:
This fix is useless. Because `debian` is set to `distro` in line 21 when it is nil.